src/thumbnailer_slope_optim.cc

```diff
@@ -177,14 +177,10 @@ Thumbnailer::Status Thumbnailer::LossyEncodeSlopeOptim(
 
 Thumbnailer::Status Thumbnailer::LossyEncodeNoSlopeOptim(
     WebPData* const webp_data) {
-  // The webp_data->size and the sum of encoded-frame sizes are inconsistent,
-  // therefore consider the bigger one as the current animation size to ensure
-  // the resulting animation size fit the byte budget.
-  int anim_size = 0;
-  for (auto& frame : frames_) {
-    anim_size += frame.encoded_size;
-  }
-  anim_size = std::max(anim_size, int(webp_data->size));
+  WebPData new_webp_data;
+  WebPDataInit(&new_webp_data);
```

Comment:
Move these two lines nearer to where `new_webp_data` is actually used (line 235)